framework/include/times/InputTimes.h

```diff
@@ -24,4 +24,10 @@ class InputTimes : public Times
 
 protected:
   virtual void initialize() override {}
+
+  /// The next external time sequences to hit
+  std::vector<Real> _input_times;
+
+  /// Storage used when 'times' isn't provided
```

Comment:
this isnt used?

Reply:
It is used in the derived class `ControllableInputTimes`